click/testing.py

```diff
@@ -260,7 +261,7 @@ def invoke(self, cli, args=None, input=None, env=None,
            The ``color`` parameter was added.
 
         :param cli: the command to invoke
-        :param args: the arguments to invoke
+        :param args: the arguments to invoke. May be an iterable or string.
```

Comment:
We need more information in the docs than that. For example, say that it will be parsed as Unix shell command, possibly mentioning `shlex`.

We need to make clear that you can't e.g. pass a string like you would write it on the Windows commandline, and that the way you write your tests using this method doesn't necessarily represent how users would call it.